DancingGoat/Scripts/PreviewMode/PreviewMode.js

```diff
@@ -0,0 +1,107 @@
+﻿const previewModeToggleClassname = 'preview-mode-switch';
+const previewModeCookieName = 'IsPreviewMode';
```

Comment:
Inconsistent naming, what's the difference between preview and preview mode?

Reply:
I don't get it. Could you more elaborate on this issue?